src/media/inner.rs

```diff
@@ -1322,10 +1321,12 @@ impl MediaInner {
         self.simulcast = Some(s);
     }
 
-    pub fn ssrc_rx_for_rid(&self, repairs: Rid) -> Option<Ssrc> {
+    pub fn find_primary_ssrc_for_rid(&self, repairs: Option<Rid>, rtx_ssrc: Ssrc) -> Option<Ssrc> {
```

Comment:
Might want to point out that if a None rid is passed in, it gives the most recent non-RTX source, which is makes it so we don't need to rely on RID header extensions.

Reply:
Well it returns the most recent non-RTX source with None as the rid as well.